src/Kernel.BrokerSupport/AccessValidatorEngine/Requests/ICheckDepartmentManagerRequest.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace DigitalOffice.Kernel.BrokerSupport.AccessValidatorEngine.Requests
+{
+  public interface ICheckDepartmentManagerRequest
+  {
+    Guid UserId { get; }
+

```

Comment:
remove empty line

Reply:
This is similar to the existing request.